modules/API/API.js

```diff
@@ -441,6 +441,27 @@ function initCommands() {
             });
             break;
         }
+        case 'get-livestream-url': {
+            const state = APP.store.getState();
+            const conference = getCurrentConference(state);
+            let livestreamUrl;
+
+            if (conference) {
+                const activeSession = getActiveSession(state, JitsiRecordingConstants.mode.STREAM);
+
+                if (activeSession?.liveStreamViewURL) {
```

Comment:
That's not what I meant :-) You can just resolve it to `activeSession?.liveStreamViewURL`, which will result in `undefined` if `activeSession` is `undefined`, so there is no need for the extra check.

Reply:
Oh, Okay. You mean like this?

const activeSession = getActiveSession(state, JitsiRecordingConstants.mode.STREAM);

livestreamUrl = activeSession?.liveStreamViewURL;

Reply:
yep

Reply:
Applied the changes and pushed.